ddl/ddl.go

```diff
@@ -134,7 +134,7 @@ type DDL interface {
 	// GetLease returns current schema lease time.
 	GetLease() time.Duration
 	// Stats returns the DDL statistics.
-	Stats() (map[string]interface{}, error)
+	Stats(vars *variable.SessionVars) (map[string]interface{}, error)
```

Comment:
@tiancaiamao Yes..Previously status variables are the same for all sessions. When selecting status variables (`Stats()`), "session" is even not passed to the function as you can see within this change. This change make it possible to support session scoped status variables.